src/build/tenscript/expression.rs

```diff
@@ -101,25 +103,24 @@ impl Parser {
         let token = self.current().clone();
         self.increment();
         match token {
-            Paren('(') => self.list(),
-            Ident(name) => Ok(Expression::Ident(name)),
-            Float(value) => Ok(Expression::Float(value)),
-            Integer(value) => Ok(Expression::Integer(value)),
-            Percent(value) => Ok(Expression::Percent(value)),
             Atom(value) => Ok(Expression::Atom(value)),
-            Token::String(value) => Ok(Expression::String(value)),
+            FloatingPoint(value) => Ok(Expression::FloatingPoint(value)),
+            Identifier(name) => Ok(Expression::Identifier(name)),
+            Parenthesis('(') => self.list(),
```

Comment:
He don't like short names for things huh. No parens or idents

Reply:
Too nerdy